README.md

```diff
@@ -208,38 +208,43 @@ Check out the Secrets section in the following READMEs:
 
 ---
 
-### Firefox Custom Profile
+### Testing
 
-**Use `npm start firefox` to start Firefox with local server configurations.**
-Available options:
+ℹ️ While it is possible to run the entire code base's test suite, in development you'll likely want to run specific test or a subset of tests. Please refer to [each package's README](#documentation) for detailed instructions on testing its respective code.
 
-- `FXA_ENV=local` or `latest` or `stable` or `stage` (NOTE: `local` is default).
-- `FXA_E10S=true` - add this flag to turn on E10S. (NOTE: `false` by default).
-- `FXA_DESKTOP_CONTEXT` - `context=` value. (NOTE: `fx_desktop_v2` is default).
-- `FIREFOX_BIN=/Applications/FirefoxNightly.app/Contents/MacOS/firefox-bin npm start`
-- `FIREFOX_DEBUGGER=true` - open [Browser Toolbox](https://developer.mozilla.org/en-US/docs/Tools/Browser_Toolbox) on start (NOTE: `false` by default for speed).
+#### Test all or some packages
 
----
+From the root directory you may test all or some FxA packages:
 
-### Functional Tests
+```bash
+# Test only `fxa-shared`
+npm test fxa-shared
 
-**The following requires [the JDK](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) and tests the local servers only.**
+# Test `fxa-auth-db-mysql` and `fxa-auth-server`
+npm test fxa-auth-db-mysql fxa-auth-server
 
-## run the following commands from the fxa-content-server directory
+# Test all packages
+npm test all
+```
 
-To run all functional tests:
```

Comment:
Was this added to the content-server readme? You could also link to content-server functional testing doc https://github.com/mozilla/ecosystem-platform/blob/master/docs/fxa-engineering/functional-testing.md.

Reply:
I have this sitting in a doc on my machine for eventual introduction to the Content Server README, but because I only tackled a handful of the READMEs here it hasn't made it back in yet. I'm happy to wait until they're all updated so we're not missing anything 🤷‍♂️

Reply:
I've updated everything, including Content Server, and it now includes both the examples removed here as well as a link to the ecosystem docs :)